Sources/BraveWallet/Crypto/Stores/NFTDetailStore.swift

```diff
@@ -65,14 +65,18 @@ class NFTDetailStore: ObservableObject {
     self.erc721Metadata = erc721Metadata
   }
   
-  func fetchMetadata() {
+  func fetchNetworkInfo() {
```

Comment:
Any reason to have these separate instead of a single `update()` function like other stores? We could still only fetch metadata if `erc721Metadata` is nil in the `update()` function.

Reply:
sure. will aggregate them into one single func.

Reply:
fixed in [314a66d](https://github.com/brave/brave-ios/pull/6617/commits/314a66d4e9565c4ac7944b0d72bf9e06aa5a5cd4)